docs/concepts/faq.md

```diff
@@ -26,6 +26,9 @@ The quickest way to get IPFS up and running on your machine is by installing [IP
 
 For installing and initializing IPFS from the command line, check out the [command-line quick start](../how-to/command-line-quick-start.md) guide.
 
+### Why doesn't my SHA hash match my CID?
```

Comment:
```suggestion
### My SHA hash is different to my CID

```

Try not to add symbols into titles. And add an empty line after a title and a paragraph.

Reply:
> Sounds like our system has a hard time with symbols

Yeah, it comes down to markdown translating headers into HTML anchors, and throwing symbols into the mix can get varied results depending on which compiler is used. So the easiest option is to remove symbols altogether.

Saying that, you're right; since this is an FAQ we should probably leave them in here, and just deal with compiler issues down the road (if they happen at all).